tests/wpt/web-platform-tests/html/semantics/embedded-content/the-img-element/null-image-source.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<meta charset="utf-8">
+<title></title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id=log></div>
+<img src="">
```

Comment:
I figured out this test for empty srcset. But, I was confused how to know if error is not thrown at a later stage?
```
async_test(function(t) {
	img = document.getElementById('srcset_id');
        setTimeout(function() {
		img.onerror = t.step_func(function(e) {
	    	       assert_unreached("empty srcset got an error instead of time out");
		});
     	        t.done();
        }, 2000);  
});
```

Reply:
The trick is to set the onerror handler immediately (and you can use `t.assert_unreached` directly instead of wrapping it in `t.step_func`), and having a `t.step_timeout` that calls `t.done()`.

Reply:
It says t.assert_unreached is not a function 😅 And setting onerror handler as assert_unreached always fails as it gets executed during assignment.

Reply:
My mistake; I meant `t.unreached_func` as described in https://web-platform-tests.org/writing-tests/testharness-api.html.